xarray/backends/pynio_.py

```diff
@@ -7,55 +7,41 @@
 import numpy as np
 
 from .. import Variable
+from .netCDF4_ import BaseNetCDF4Array
 from ..core.utils import FrozenOrderedDict, Frozen, NDArrayMixin
 from ..core import indexing
 
 from .common import AbstractDataStore, DataStorePickleMixin
 
 
-class NioArrayWrapper(NDArrayMixin):
```

Comment:
I inherited from `BaseNetCDF4Array` instead to resuse the code for simplicity.  This seemed like redundant code.